lib/submit-request.js

```diff
@@ -290,6 +299,20 @@ SubmitRequest.prototype._shouldSaveMilestoneSnapshot = function(snapshot) {
   return this.saveMilestoneSnapshot;
 };
 
+SubmitRequest.prototype._fetchCreateOpVersion = function(callback) {
+  var create = this.snapshot.m._create;
```

Comment:
Note that this approach has one pitfall: it opens up a *new* corner case where
an op may be incorrectly rejected instead of acked:

 1. Client A submits Create 1
 2. Create 1 committed
 3. Client A disconnects before receiving the ack
 4. Client B submits and commits Delete, then Create 2
 5. Client A reconnects, and resubmits Create 1
 6. Create 1 doesn't match the current snapshot metadata for Create 2, so it's rejected with `DOC_ALREADY_CREATED` instead of the non-fatal `OP_ALREADY_SUBMITTED`

I don't know how much we care about this? It definitely is a bug that didn't used to exist, but I'm not sure it's important enough to keep leaning on `getCommittedOpVersion()` all the time? 

In my personal opinion, if two clients are creating/deleting/recreating the same `Doc` in such a close span of time, errors are bound to happen anyway, since creates/deletes don't have great transformation behaviour. And if this is an expected use case, consumers should already have robust error-handling in place for this sort of conflict.

Reply:
Per discussion, this should be a vanishingly small edge case in practice, and Client A reconnecting may run into other errors while reconnecting anyways that would require it to re-initialize.